src/typography/Text/Text.tsx

```diff
@@ -62,6 +99,7 @@ export const Text = ({
             weightMap[weight],
             sizeMap[size],
             colorMap[color],
+            boxColorMap[boxColor],
```

Comment:
I think that default option being empty string is confusing. Can we just make the property optional and have `boxColor && boxColorMap[boxColor]` on line 102 ?

Reply:
No problem